cmd/minikube/cmd/service_list.go

```diff
@@ -41,24 +46,37 @@ var serviceListCmd = &cobra.Command{
 			exit.WithError("Error getting client", err)
 		}
 		defer api.Close()
+		profileName := viper.GetString(pkg_config.MachineProfile)
+		if !machine.IsHostRunning(api, profileName) {
+			exit.WithCodeT(exit.Unavailable, "profile {{.name}} is not running. ", out.V{"name": profileName})
+		}
 		serviceURLs, err := service.GetServiceURLs(api, serviceListNamespace, serviceURLTemplate)
 		if err != nil {
 			out.FatalT("Failed to get service URL: {{.error}}", out.V{"error": err})
 			out.ErrT(out.Notice, "Check that minikube is running and that you have specified the correct namespace (-n flag) if required.")
 			os.Exit(exit.Unavailable)
 		}
+		cfg, err := config.Load(viper.GetString(config.MachineProfile))
+		if err != nil {
+			exit.WithError("Error getting config", err)
+		}
 
 		var data [][]string
 		for _, serviceURL := range serviceURLs {
 			if len(serviceURL.URLs) == 0 {
 				data = append(data, []string{serviceURL.Namespace, serviceURL.Name, "No node port"})
 			} else {
-				data = append(data, []string{serviceURL.Namespace, serviceURL.Name, strings.Join(serviceURL.URLs, "\n")})
+				data = append(data, []string{serviceURL.Namespace, serviceURL.Name, "", strings.Join(serviceURL.URLs, "\n")})
+
 			}
 
 		}
-
 		service.PrintServiceList(os.Stdout, data)
+		if runtime.GOOS == "darwin" && cfg.Driver == oci.Docker {
+			out.FailureT("Accessing service on docker driver for mac is not fully implemented yet.\nTo read about workarroudns on mac and update on the work in progress, please reffer to this link:\nhttps://github.com/kubernetes/minikube/issues/6778")
```

Comment:
"Accessing service is not implemented yet for docker driver on Mac.\nThe following issue is tracking the in progress work: